File: Web.Admin/src/api/services/Center/clientUser/models/QueryClientUserPagedInput.ts
import type { PagedInput } from "fast-element-plus";
import type { ClientUserTypeEnum } from "@/api/enums/ClientUserTypeEnum";
import type { GenderEnum } from "@/api/enums/GenderEnum";

/**
 * 获取客户端用户分页列表输入
 */
export interface QueryClientUserPagedInput extends PagedInput  {
	/**
	 * 应用Id
	 */
	appId?: number;
	/**
	 * 
	 */
	userType?: ClientUserTypeEnum;
	/**
	 * 
	 */
	sex?: GenderEnum;
	/**
	 * 
	 */
	readonly isOrderBy?: boolean;
}

File: Api.Server/src/Center.Service/ClientUser/Dto/QueryClientUserPagedInput.cs
// Copyright © 2018-Present 小方
// SPDX-License-Identifier: Apache-2.0
// 
// 本文件依据 Apache License 2.0 授权，完整条款见仓库根目录 LICENSE。
// 本软件按“原样”提供，相关免责声明及责任限制以许可证及适用法律为准。
// 版权来源、合法使用与二次开发责任说明见仓库根目录 README.md。

using Fast.Center.Domain;

namespace Fast.Center.Service.ClientUser.Dto;

/// <summary>
/// 获取客户端用户分页列表输入
/// </summary>
public class QueryClientUserPagedInput : PagedInput
{
    /// <summary>
    /// 应用Id
    /// </summary>
    public long? AppId { get; set; }

    /// <summary>
    /// 用户类型
    /// </summary>
    public ClientUserTypeEnum? UserType { get; set; }

    /// <summary>
    /// 性别
    /// </summary>
    public GenderEnum? Sex { get; set; }
}
